settings.py

```diff
@@ -496,6 +496,9 @@
     ]
 }
 
+# Note: if you change this, make sure to update the cronjob in commcarehq-ansible
+RESTORE_PAYLOAD_DIR = os.path.join(FILEPATH, 'restore_payloads')
```

Comment:
how do we feel about putting these in the code root directory? it somehow feels wrong to me, though i don't think i can completely articulate why other than from some abstract "separation of concerns" perspective. @snopoke @benrudolph @dannyroberts